### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by pallavi**
What: Set up RESTful API sfmovies as a Node package using the Yeoman Lob Generator and the Hapi framework.

Why: This sets up a server and is the base for building the API.

Details:
- Generated a package.json file and defined an entry point
- Set up basic structure of Node Package using Yeoman Lob Generator, set up config files
- Created a server using Hapi that connects to port specified in config files
- Defined a start command using nodemon that updates whenever there is a change in our code
- Generated a npm-shrinkwrap.json file that locks down the versions of all the dependencies

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+node_js:
+  - 5
+env:
+  - NODE_ENV="test"
+#before_script:
+  #- npm run db:setup:user
+  #- npm run db:reset
+script:
+  - npm test
+  #- npm run enforce
```
Comment (elnaz) [inline .travis.yml]: minor but probably remove the lines entirely until you're ready to add them back in

File: config/development.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  PORT : 3000
```
Comment (elnaz) [inline config/development.js]: extra space after `PORT`

File: lib/server.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Hapi = require('hapi');
+
+const Config = require('../config');
+
+const server = new Hapi.Server({
+  connections: {
+    router: { stripTrailingSlash: true } //removes trailing slashes on incoming paths
```
Comment (elnaz) [inline lib/server.js]: remove comment

**Comment by elnaz** [comment]
Just small stuff, LGTM!

**Comment by mgartner** [comment]
LGTM 💯

**Pull request merged**